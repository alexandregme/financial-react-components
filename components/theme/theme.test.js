import React from 'react';
import { shallow } from 'enzyme';
import Theme from 'components/theme';

describe('Theme', () => {
  let mountedTheme;

  beforeEach(() => {
    mountedTheme = shallow(<Theme />);
  });

  it('renders without crashing', () => {
    shallow(<Theme />);
    expect(mountedTheme).toMatchSnapshot();
  });

  it('expect to have Layout component', () => {
    expect(mountedTheme.find('div')).toHaveLength(1);
  });
});
