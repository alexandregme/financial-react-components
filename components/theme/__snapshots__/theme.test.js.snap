// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme renders without crashing 1`] = `
<Fragment>
  <div />
</Fragment>
`;
